src/encoded/static/components/cart/cart.js

```diff
@@ -93,53 +97,182 @@ CartSearchResults.defaultProps = {
 
 
 /**
- * Display one term of the File Format facet.
+ * Display a count of the total number of files selected for download given the current facet term
+ * selections. Shows the facet-loading progress bar if needed.
+ */
```

Comment:
I like all of the visual cues on this component to let you know when everything is done loading / when the number of files is updated.